addon/components/carousel-arrow.js

```diff
@@ -1,32 +1,29 @@
-import Ember from 'ember';
-import layout from '../templates/components/carousel-arrow';
+import Component from 'ember-component';
+import layout    from '../templates/components/carousel-arrow';
+
+import computed from 'ember-computed';
```

Comment:
Well in the RFC they say the following 

> For those who have already adopted modules via the `ember-cli-shims` package, we
 will provide a migration tool to rewrite shim modules into the final module API. 

And since it is not 100% sure how the new system will work, I think this is the best way to prepare for that, not 'ember-new-modules-shim'